pkg/parse/state.go

```diff
@@ -98,12 +99,7 @@ const retryLimit = 12
 //	16.295984061s, 34.325711987s, 1m5.465642392s, 2m18.625713221s,
 //	4m24.712222056s, 9m18.97652295s, 17m15.344384599s, 35m15.603237976s.
 func defaultBackoff() wait.Backoff {
-	return wait.Backoff{
-		Duration: 1 * time.Second,
-		Factor:   2,
-		Steps:    retryLimit,
-		Jitter:   0.1,
-	}
+	return util.BackoffWithDurationAndStepLimit(0, retryLimit)
```

Comment:
Hmm. What happened to RetryPeriod? Isn't the base duration supposed to be configurable?

The reconciler.Options.RetryPeriod get passed to the parser.Options.RetryPeriod, but then never used?

Reply:
Appears it's used in a few places:
- https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/pkg/parse/run.go#L92
- https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/pkg/parse/run.go#L106
- https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/pkg/parse/run.go#L162
- https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/pkg/parse/run.go#L329